app/src/main/java/com/lagradost/cloudstream3/plugins/PluginManager.kt

```diff
@@ -163,10 +163,10 @@ object PluginManager {
         return getKey(PLUGINS_KEY_LOCAL) ?: emptyArray()
     }
 
-    private val CLOUD_STREAM_FOLDER =
-        Environment.getExternalStorageDirectory().absolutePath + "/Cloudstream3/"
+    private val CLOUDSTREAM3_FOLDER =
+        Environment.getExternalStorageDirectory().absolutePath + "/CloudStream3/"
```

Comment:
"Linux file system treats file and directory names as case-sensitive" so this will break all current plugins that are located in "Cloudstream3". Loop all folders that match Cloudstream3 in a case insensitive manner if you want to solve this.

Reply:
Makes sense.